dataeng/jobs/analytics/LoadPaypalCaseReportToVertica.groovy

```diff
@@ -0,0 +1,27 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+
+class LoadPaypalCaseReportToVertica{
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("load-paypal-casereport-to-vertica"){
+            logRotator common_log_rotator(allVars)
+            parameters common_parameters(allVars)
+            parameters {
+                stringParam('SCHEMA', allVars.get('SCHEMA'))
+                stringParam('RUN_DATE', allVars.get('RUN_DATE', 'yesterday'), 'Run date for the job. A string that can be parsed by the GNU coreutils "date" utility.')
```

Comment:
nit:  maybe say something like "The date for which to pull and load case report" instead of "run date".   Latter suggests to me it might be when the job is actually run.

Reply:
ok